cli/args/flags.rs

```diff
@@ -3917,6 +3917,8 @@ fn permission_args(app: Command, requires: Option<&'static str>) -> Command {
                                              <p(245)>--deny-import  |  --deny-import="example.com:443,github.com:443"</>
       <g>DENO_TRACE_PERMISSIONS</>               Environmental variable to enable stack traces in permission prompts.
                                              <p(245)>DENO_TRACE_PERMISSIONS=1 deno run main.ts</>
+      <g>DENO_PERMISSIONS_AUDIT</>               Environmental variable to generate a JSONL file with all permissions accesses.
```

Comment:
Bikeshedding time: should this be `DENO_AUDIT_PERMISSIONS`? Should we maybe need 1 env var for activating this feature that dumps it to a file and another to specify a file (if not specified uses the default one)

Reply:
yea was considering `DENO_AUDIT_PERMISSIONS` too.
regarding multiple env vars: how about `1` or `true` uses a default path, and any other value gets treated as a path and use that instead?